frontend/src/libs/helpers/get-first-letters/get-first-letters.helper.ts

```diff
@@ -1,8 +1,14 @@
+import {
+  EMPTY_STRING,
+  ONE_SPACE_DELIMITER,
+  ZERO_COUNT,
+} from '~/libs/constants/constants.js';
+
 const getFirstLetters = (initialString: string): string => {
   return initialString
-    .split(' ')
-    .map((word) => word.charAt(0))
-    .join('');
+    .split(ONE_SPACE_DELIMITER)
```

Comment:
In this case it's more clear to understand explicitly what is used as delimeter,, then to search for a constant to find out what does it mean
```suggestion
    .split(' ')
```